native/src/main/java/io/ballerina/stdlib/io/nativeimpl/DataChannelUtils.java

```diff
@@ -125,8 +124,8 @@ public static Object readFloat32(BObject dataChannelObj) {
         try {
             return channel.readDouble(Representation.BIT_32);
         } catch (IOException e) {
-            log.error("error occurred while reading Float32", e);
-            return IOUtils.createError(e);
+            String message = "error occurred while reading Float32: " + e.getMessage();
```

Comment:
We just need to remove the loggers. We can keep the error messages as it is

Reply:
Make the necessary changes everywhere

Reply:
Ok

Reply:
@sahanHe changes done at all places as suggested.